Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -479,6 +490,42 @@ static void webkit_website_data_manager_class_init(WebKitWebsiteDataManagerClass
             nullptr, nullptr,
             FALSE,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY)));
+
+    /**
+     * WebKitWebsiteDataManager:origin-storage-ratio:
+     *
+     * The ratio which is used to calculate the origin data storage based on disk space.
```

Comment:
I still think we need to explain this better. If I've understood it correctly this value is the percentage of volume space that can be used for data storage for every domain. Right? Because the ration to calculate makes me wonder, but how is it calculated, how can I decide a value if I don't know how it's calculated?